main.py

```diff
@@ -5,6 +5,8 @@
 from fastapi import FastAPI, Request, Response
 from dotenv import load_dotenv
 import os
+from schemas.helper import *
+from schemas.schema import WebhookRequestData, UserInfo
```

Comment:
Normally I will do this:
add this to __init__.py:
`import sys
# setting path

sys.path.append('..')
from .helper import *
from .schema import *`

Then import the func or class you want by:
`from schemas import WebhookRequestData, UserInfo...`
or simply use `from schemas import *`